app/assets/stylesheets/two-factor-authentication.scss

```diff
@@ -0,0 +1,5 @@
+// Styles around two-factor authentication
+
+.two-factor-qr {
+  background-color: #ffffff;
```

Comment:
- Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.
- Color `#ffffff` should be written as `#fff`

Reply:
We have $white in our variables already ;)